apis/iam/v1beta1/diff_test.go

```diff
@@ -0,0 +1,856 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
```

Comment:
Consider adding a comment here explaining why this sorting is necessary. It helps future maintainers understand that the order of diffs is not guaranteed and needs to be normalized for stable test comparisons.